zh-CN/mapping_dev.zh_CN

```diff
@@ -1342,392 +1342,392 @@ dlgSgRenameFile.inf=输入文件的新路径和名称。
+dlgSgRepositoryFrameCloseWithoutPush.tle=关闭
+dlgSgRepositoryRemove.btn"Remove"=删除
+dlgSgRepositoryRemove.chk=不再显示
+dlgSgRepositoryRemove.fur=这会将仓库保留在磁盘上，但只是让 SmartGit 忘记它们。
+dlgSgRepositoryRemove.hdl%1=您确定要删除 $1 的仓库吗？
+dlgSgRepositoryRemove.tle=删除
 dlgSgRepositorySettings.chk"Always fetch new commits, tags and branches from submodule"=\
- Always fetch new commits, tags and branches from submodule
-dlgSgRepositorySettings.chk"Initialize new submodules"=Initialize new submodules
-dlgSgRepositorySettings.chk"Prune obsolete remote tracked branches"=Prune obsolete remote tracked branches
+ 始终从子模块中获取新的提交，标记和分支
+dlgSgRepositorySettings.chk"Initialize new submodules"=初始化新的子模块
+dlgSgRepositorySettings.chk"Prune obsolete remote tracked branches"=删除过时的远程跟踪分支
 dlgSgRepositorySettings.chk"Remember as default"=记住默认
 dlgSgRepositorySettings.chk"Sign all commits"=签名所有提交
 dlgSgRepositorySettings.chk"Update registered submodules"=更新已注册的子模块
 dlgSgRepositorySettings.edt"Email"=电子邮箱
-dlgSgRepositorySettings.edt"GPG Program"=GPG Program
+dlgSgRepositorySettings.edt"GPG Program"=GPG 程序
 dlgSgRepositorySettings.edt"Name"=用户名
-dlgSgRepositorySettings.edt"Signing Key"=Signing Key
-dlgSgRepositorySettings.edt"Submodule handling\*"=Submodule handling\*
+dlgSgRepositorySettings.edt"Signing Key"=签名密钥
+dlgSgRepositorySettings.edt"Submodule handling\*"=子模块处理\*
 dlgSgRepositorySettings.edt"Text File Encoding"=文本编码
-dlgSgRepositorySettings.hdl=编辑有效的版本库设置
-dlgSgRepositorySettings.inf=在这里，您可以查看和编辑用户配置文件和版本库的有效Git设置。要将选项保存为用户设置，请选择“记住默认”。
-dlgSgRepositorySettings.tle=版本库设置
+dlgSgRepositorySettings.hdl=编辑有效的仓库设置
+dlgSgRepositorySettings.inf=在这里，您可以查看和编辑用户配置文件和仓库的有效 Git 设置。要将选项保存为用户设置，请选择 “记住默认” 。
+dlgSgRepositorySettings.tle=仓库设置
 dlgSgResetAdv.btn"Reset"=重置
 dlgSgResetAdv.chk"Thoroughly fix line endings according to .gitattributes"=\
- Thoroughly fix line endings according to .gitattributes
-dlgSgResetAdv.hdl%1=Reset to commit $1
-dlgSgResetAdv.inf=Reset the current branch head to the selected commit and optionally update Index and working tree.
+ 根据 .gitattributes 彻底修复行尾
+dlgSgResetAdv.hdl%1=重置提交 $1
+dlgSgResetAdv.inf=将当前分支 HEAD 重置为所选提交，并可选择更新索引和工作树。
 dlgSgResetAdv.rbt"Don't touch the Index nor the working tree - 'soft'"=\
- Don't touch the Index nor the working tree - 'soft'
+ 不要触摸索引或工作树 - “软”
 dlgSgResetAdv.rbt"Reset the Index and the working tree - 'hard'"=\
- Reset the Index and the working tree - 'hard'
+ 重置索引和工作树 - “硬”
 dlgSgResetAdv.rbt"Reset the Index and the working tree, skipping modified files - 'keep'"=\
- Reset the Index and the working tree, skipping modified files - 'keep'
+ 重置索引和工作树，跳过修改过的文件 - “保留”
 dlgSgResetAdv.rbt"Reset the Index and the working tree, skipping working tree modified files - 'merge'"=\
- Reset the Index and the working tree, skipping working tree modified files - 'merge'
+ 重置索引和工作树，跳过工作树修改的文件 - “合并”
 dlgSgResetAdv.rbt"Reset the Index but not the working tree - 'mixed'"=\
- Reset the Index but not the working tree - 'mixed'
+ 重置索引，但不重置工作树 - “混合”
```

Comment:
For the original English texts, we won't add '--' because the terms are pretty well-known for command line users (for which we have added this 'soft', 'hard', ... information).

Reply:
I agree to use "--hard"

Reply:
> For the original English texts, we won't add '--' because the terms are pretty well-known for command line users (for which we have added this 'soft', 'hard', ... information).

If translate to Chinese, Just keep 'hard' without '--', will make it hard to understand.

Reply:
> If translate to Chinese, Just keep 'hard' without '--', will make it hard to understand.

OK, then let's add '--' for the translation.